builds/misc/packages-release.yaml

```diff
@@ -435,5 +435,138 @@ stages:
+        displayName: Get secrets
+        inputs:
+          azureSubscription: $(az.subscription)
+          keyVaultName: $(kv.name)
+          secretsFilter: >-
+            IotEdge1-PAT-msazure
+
+      - bash: |
+          # Source the scripts & Update version files
+          sudo chmod +x $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+          source $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+
+          wait-for-dpkg-lock 120
+
+          if ! command -v jq &> /dev/null
+          then
+              sudo apt-get install jq -y
+          fi
+        displayName: Setup Test Agent
+        name: SetupTestAgent
+
+      - bash: |
+          # Fetch versions to be used for each component
+          edgeletVersion=$(cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-edge.json | jq ".\"aziot-edge\"" | tr -d '"')
+          iisVersion=$(cat cat $(Build.SourcesDirectory)/azure-iotedge/latest-aziot-identity-service.json | jq ".\"aziot-identity-service\"" | tr -d '"')
+
+          echo "##vso[task.setvariable variable=edgeletVersion;isOutput=true]$edgeletVersion"
+          echo "##vso[task.setvariable variable=iisVersion;isOutput=true]$iisVersion"
+        env:
+          GITHUB_PAT: "$(GitHubAccessToken)"
+        displayName: Get Parameters From Azure-iotedge
+        name: GitHubParameters
+
+      - bash: |
+          edgeletVersion=$(GitHubParameters.edgeletVersion)
+          iisVersion=$(GitHubParameters.iisVersion)
+
+          wget https://github.com/Azure/azure-iotedge/releases/download/$edgeletVersion/aziot-edge_$edgeletVersion-1_$(os)_$(arch).$(ext) -O $(System.ArtifactsDirectory)/aziot-edge_$edgeletVersion-1_$(os)_$(arch)_github.$(ext)
+          wget https://github.com/Azure/azure-iotedge/releases/download/$edgeletVersion/aziot-identity-service_$iisVersion-1_$(os)_$(arch).$(ext) -O $(System.ArtifactsDirectory)/aziot-identity-service_$iisVersion-1_$(os)_$(arch)_github.$(ext)
+        displayName: Download Edgelet GitHub
+
+      - bash: |
+          # Source the scripts & Update version files
+          sudo chmod +x $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+          source $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+
+          if [[ "$(os)" == "ubuntu"* ]]; then
+          # Ubuntu
+            if [[ "$(os)" == *"20.04" ]]; then
+              setup-focal-source-apt
+              setup-config-apt "https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb"
+            elif [[ "$(os)" == *"18.04" ]]; then
+              setup-config-apt "https://packages.microsoft.com/config/ubuntu/18.04/multiarch/packages-microsoft-prod.deb"
+            else
+              echo "Unsupported OS: $(os)"
+              exit 1;
+            fi
+          elif [[ "$(os)" == "debian11" ]]; then
+            setup-config-apt "https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb"
```

Comment:
Does the `setup-config-apt` function in smokeTestHelper.sh need to be updated at all to handle debian? I noticed the instructions to register the microsoft repo are slightly different on ubuntu vs. debian...

Reply:
It is. I think the usual instructions for ubuntu AMD64 should work fine with the ARM. Since ARM is not enabled right now, I'm going to let my future self deal with the debian repo sourcing later. I suspect with the 1ES changes, there are more steps to be done than just this simple download-and-install (unfortunately).